easybuild/easyconfigs/m/Minimac/Minimac-2013.7.17-goolf-1.4.10.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
+
+name = 'Minimac'
+version = '2013.7.17'
+versionsuffix = '-beta'
+
+homepage = 'http://genome.sph.umich.edu/wiki/Minimac'
+description = """ Minimac is a low memory, computationally efficient implementation 
+ of the MaCH algorithm for genotype imputation. It is designed to work on phased genotypes
+ and can handle very large reference panels with hundreds or thousands of haplotypes. 
+ The name has two parts. The first, 'mini', refers to the modest amount of computational
+ resources it requires. The second, 'mac', is short hand for MaCH, our widely used 
+ algorithm for genotype imputation.  """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+# the source file doesn't include version in the name so probably you should check
+# in their website if the version number and date is correct
+source_urls = ['http://www.sph.umich.edu/csg/cfuchsb/']
+sources = ['%(namelower)s.src.tgz']
```

Comment:
include datestamp/version in source tarball name (breaks auto-downloading, but that's not a big issue imho)

```python
# download minimac.src.tgz from http://www.sph.umich.edu/csg/cfuchsb and rename
sources = ['%(namelower)s-%(version)s.src.tgz']
```